Cargo.toml

```diff
@@ -101,6 +115,8 @@ incremental = true
 criterion = { version = "0.4.0", features = ["async_tokio", "html_reports"] }
 diff = "0.1.13"
 matches = "0.1.9"
-netns-rs = "0.1.0"
 test-case = "3.0.0"
 #debug = true
+
+[target.'cfg(target_os = "linux")'.dev-dependencies]
```

Comment:
nit: do we need this? since its already a core `dependency`?

Reply:
nope ... removed.